pressbooks/pressbooks-book-directory-fe
# Improve site meta for SEO purposes

This PR solves #313 #307 and #308 

This includes meta tags required to have a proper setup for SEO purposes and responsiveness.
This includes the robots.txt for non productions environments to avoid indexing.

How to test

1. Download and test the app everything should work as expected
2. Run the build process to see if the dist/index.html is generated according to the required settings
3. Create a `VITE_APP_GA_MEASUREMENT_ID` in your .env and review that value is added to the `dist/index.html` after running `npm run build`
